includes/Documents/PackingSlip.php

```diff
@@ -201,11 +201,50 @@ public function init_settings() {
 			$settings_fields = WPO_WCPDF()->settings->move_setting_after_id( $settings_fields, $pro_notice, 'enabled' );
 		}
 
-		// allow plugins to alter settings fields
+		// Legacy filter to allow plugins to alter settings fields.
 		$settings_fields = apply_filters( 'wpo_wcpdf_settings_fields_documents_packing_slip', $settings_fields, $page, $option_group, $option_name );
```

Comment:
Maybe this filter has to be changed?

Reply:
Do you mean using `apply_filters_deprecated()`?

Reply:
No, just append the output format.

Reply:
Since this is the legacy filter, should we append the output format?
If we're going to change the filter name, then we can totally remove it, I think?

Reply:
I thought it was already deprecated. Perhaps we should address it in a separate issue.